src/components/InputTodo.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+//  eslint-disable-next-line
```

Comment:
I think it will be better if there is no custom changes to linter config in order to meet project requirements.